task/src/main/java/org/exoplatform/platform/qa/ui/task/pageobject/ManagementLabels.java

```diff
@@ -0,0 +1,50 @@
+package org.exoplatform.platform.qa.ui.task.pageobject;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.selenium.TestBase;
+import org.exoplatform.platform.qa.ui.selenium.testbase.ElementEventTestBase;
+
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.*;
+import static com.codeborne.selenide.Selectors.*;
+
+
+/**
+ * This class will define actions about management tasks
+ */
+
+public class ManagementLabels {
+  private final TestBase       testBase;
+
+  private ElementEventTestBase evt;
+
+  public ManagementLabels(TestBase testBase) {
+    this.testBase = testBase;
+    this.evt = testBase.getElementEventTestBase();
+  }
+public void addLabel(String label){
```

Comment:
Indentation + add blank line (to do for the whole class). Use code formatter (see [developer guide](http://developer.exoplatform.org/))